examples/lb3-application/README.md

```diff
@@ -103,6 +103,242 @@ Start the new LB4 application
 Open the URL printed to console to view your front-end served from `public`
 directory or go to `http://127.0.0.1:3000/explorer` to explore the REST API.
 
+The LB3 application is now successfully mounted on a LB4 application, but we
+can further optimize the setup so that only the bare neccessary artifacts
+from the LoopBack 3 application remain. This includes moving almost all of the
+middleware to a common location so that they are shared by both the LoopBack 3
+and the LoopBack 4 apps.
+
+## Migrating Express middleware from LB3 app
+
+First off, edit the LB3 app's `config.json` file.
+
+Remove these properties, they are not required anymore:
+
+```json
+"host": "0.0.0.0",
+"port": 3000,
+```
+
+Change `restApiRoot` to `/` or any path where you would like the LB3 app to be
+mounted; that's relative to the LB4 app's REST API path, which defaults to
+`/api`.
+
+And then add `"handleUnknownPaths": false` to the `rest` property, this will
+prevent the LB3 REST api from sending a 404 response for requests it cannot
+handle.
+
+The `config.json` file should now look like this:
```

Comment:
Is it a good idea to repeat the content of config.json file in README? Can we replace this long code snippet with a [link](https://github.com/strongloop/loopback-next/blob/master/examples/lb3-application/lb3app/server/config.json) to the file on GitHub instead? That way there is only one place to maintain. 

Same comment applies to other long code snippets, e.g. `middleware.json` and so on.

Reply:
It is distracting having to click away to see the something that should ideally be flowing with the content.